src/v2/Components/v2/ArtworkFilter/ArtworkFilters/__tests__/MaterialsTermFilter.jest.tsx

```diff
@@ -78,5 +78,30 @@ describe("MaterialsTermFilter", () => {
       expect(context.filters.materialsTerms).toContain("oil")
       expect(context.filters.materialsTerms).toContain("canvas")
     })
+
+    it("autocompletes available options", async () => {
+      const wrapper = getWrapper({
+        aggregations: [
+          {
+            slice: "MATERIALS_TERMS",
+            counts: [
+              {
+                name: "Oil",
+                count: 10,
+                value: "oil",
+              },
+              {
+                name: "Canvas",
+                count: 5,
+                value: "canvas",
+              },
+            ],
+          },
+        ],
+      })
+
+      expect(wrapper.find("input")).toBeDefined()
+      // returns 0 nodes 🤔
```

Comment:
Wanted to get a unit test around this, but struggling to get Enzyme to do what I want. I figured a full `mount` would produce the entire component subtree here, but apparently not?

Since this is already tested via the FacetAutosuggest spec, I'm fine ditching this, but figured I'd be a good sport first. Does anything stand out here?